pjmedia/include/pjmedia/stream.h

```diff
@@ -298,6 +298,25 @@ PJ_DECL(pjmedia_transport*) pjmedia_stream_get_transport(pjmedia_stream *st);
 PJ_DECL(pj_status_t) pjmedia_stream_start(pjmedia_stream *stream);
 
 
+/**
+ * Modify the stream's codec parameter after the codec is opened.
+ * Note that not all codec parameters can be modified during run-time.
+ * When the parameter cannot be changed, this function will return
+ * non-PJ_SUCCESS, and the original parameters will not be changed.
```

Comment:
From a quick check, non-Opus codecs will just ignore parameter changes other than VAD/PLC and still return `PJ_SUCCESS`.